react/features/base/conference/middleware.any.ts

```diff
@@ -2,6 +2,9 @@ import { AnyAction } from 'redux';
 
 // @ts-ignore
 import { MIN_ASSUMED_BANDWIDTH_BPS } from '../../../../modules/API/constants';
+// eslint-disable-next-line lines-around-comment
+// @ts-expect-error
+import VideoLayout from '../../../../modules/UI/videolayout/VideoLayout';
```

Comment:
This will break things on mobile. VideoLayout ends up importing react-dom through LargeVideoManager.